Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -29,9 +31,10 @@ class MyChannelEndpoint(MyChannelBaseEndpoint):
     """
 
     def getChild(self, path, request):
-        child_handler_dict = {"overview": MyChannelOverviewEndpoint, "torrents": MyChannelTorrentsEndpoint}
+        child_handler_dict = {"overview": MyChannelOverviewEndpoint, "torrents": MyChannelTorrentsEndpoint,
```

Comment:
Why aren't you using `putChild()` instead? https://twistedmatrix.com/trac/browser/tags/releases/twisted-14.0.0/twisted/web/resource.py#L209